JMeterAKSPipeline/LoadTestInfra.yml

```diff
@@ -0,0 +1,20 @@
+pool:
+  vmImage: "windows-latest"
+
+trigger: none
+
```

Comment:
define variable sections and comment all the variables being used in the pipeline for reference.

Reply:
Is it a valid practice or shall we mention it in README.

Reply:
adding variables names would be just for reference so that user won't have to dig through references of variables used in pipeline or go through readme.md. 
Also provides additional advantage that user can simply edit yml file and assign the values to variables by uncommenting

Reply:
done